test/conversions.js

```diff
@@ -430,22 +430,78 @@ const tests = {
 					expect: [1.0, 0.0, null],
 				},
 				{
-					name: "sRGB red (D65) to OKlab",
+					name: "sRGB red (D65) to OKlch",
 					args: "red",
 					expect: [0.6279553639214311, 0.2576833038053608, 29.23388027962784],
 				},
 				{
-					name: "sRGB lime (D65) to OKlab",
+					name: "sRGB lime (D65) to OKlch",
 					args: "lime",
 					expect: [0.8664396175234368, 0.2948272245426958, 142.4953450414439],
 				},
 				{
-					name: "sRGB blue (D65) to OKlab",
+					name: "sRGB blue (D65) to OKlch",
 					args: "blue",
 					expect: [0.45201371817442365, 0.3132143886344849, 264.0520226163699],
 				},
 			],
 		},
+		{
+			name: "OKLrab",
+			data: {
+				toSpace: "oklrab",
+			},
+			tests: [
+				{
+					name: "sRGB white (D65) to OKlrab",
+					args: "white",
+					expect:  [ 1.0000000000000002, -4.996003610813204e-16, 0 ],
```

Comment:
Should this actually be zero and this is cumulative error?

Reply:
Yeah, cumulative floating point error.

Oklab in Color.js currently yields that for white, so OkLrab will also yield that for white as OkLrab just applies a toe function to the lightness, and at white, there is no change.

```js
> new Color('white').to('oklab').coords
[ 1.0000000000000002, -4.996003610813204e-16, 0 ]
```

Floating point errors can vary slightly depending on how the original matrix was generated, the exact steps and in what order. The invert matrix result can be slightly different (floating point error-wise) depending on exactly how the inversion is performed (did it use LU decomposition, some other way, what order were operations done in, etc).

This isn't too different for other Lab-like spaces, especially those that employ more complicated calculations:

```js
> new Color('white').to('ictcp').coords
[ 0.5806888810416109, 1.1102230246251565e-16, 2.914335439641036e-16 ]
```

I think my library has slightly different floating point errors. It yields an exact 1 for lightness, but non-zero results for both a and b.

```py
>>> Color('white').convert('oklab').coords()
[1.0, -4.996003610813204e-16, 1.1102230246251565e-16]
```
But the same things happen for various other Lab-ish spaces (Y is the lightness in XYB):

```py
>>> Color('white').convert('hunter-lab').coords()
[100.0, 1.912989643463523e-14, -7.46260389005987e-15]
>>> Color('white').convert('xyb').coords()
[0.0, 0.8453085619621621, 2.220446049250313e-16]
```